components/bio-formats-tools/test/loci/formats/tools/ImageConverterTest.java

```diff
@@ -216,4 +218,20 @@ public void testSPWSeries() throws FormatException, IOException {
     };
     assertConversion(args);
   }
+
+  @Test
+  public void testCrop() throws FormatException, IOException {
+    outFile = tempDir.resolve("test.ome.tiff").toFile();
+    String[] args = {
+      "-tilex", "128", "-tiley", "128",
+      "-crop", "256,256,256,256", "test.fake", outFile.getAbsolutePath()};
```

Comment:
Maybe could also include a test that isn't exactly on tile boundaries.

Reply:
Added in 7a11de2

Reply:
:+1: